sql/moz-fx-data-marketing-prod/iprospect/adspend_v1/metadata.yaml

```diff
@@ -0,0 +1,18 @@
+friendly_name: Adspend
+description: >
+  Import detailed adspend iProspect CSV data from
+  moz-fx-data-marketing-prod-iprospect storage bucket.
+labels:
+  incremental: true
+owners:
+  - ascholtz@mozilla.com
+scheduling:
```

Comment:
A depends on past condition, maybe? It looks like data from newer dates should be prioritized over older values. On the other hand, a missed day doesn't block the next day since there is a 30 window from the import.

Reply:
Yeah, I don't think it is strictly necessary in this case. Not having `depends_on_past` would have the advantage of running potential backfills in Airflow without having to reduce the number of parallel task runs.